docs/src/content/concepts-options.md

```diff
@@ -34,7 +34,10 @@ persistent by saving the settings out to a YAML configuration file (please see
 the specific tool's interactive help for details on how to do this).
 
 For all tools, options can be set directly by name using the `--set`
-command-line option. Please see the command-line help (`--help`) for usage.
+command-line option. Please see the command-line help (`--help`) for usage. Example on Windows:
+```
+mitmdump.exe -p 8088 --set ssl_version_client=all --set ssl_version_server=all
```

Comment:
Can we please make this not specific to a singel platform, and keep it as concise as possible (i.e. either demonstrate a single option, or demonstrate a single list option with two settings)?

Reply:
For example:
```
mitmweb --set  anticomp=true
mitmproxy --set  allow_hosts=[a.b.c.d, e.f.g.h]
```
?

Reply:
Corrected. Maybe `<IPv4-1>`, `<IPv4-2>`, `<IPv6-1>`, `<IPv6-2>` would be better, just to say we can mix both?